src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -7612,6 +7597,88 @@ interface CorporationInfo {
   divisions: Division[];
 }
 
+/**
+ * Corporation related constants
+ * @public
+ */
+interface CorpConstants {
+  /** Name of the corporation */
+  states: string[];
+  /** Funds available */
+  unlocks: string[];
+  /** Revenue per second this cycle */
+  upgrades: string[];
+  /** Expenses per second this cycle */
+  researches: Record<string, string[]>;
+  /** Indicating if the company is public */
+  jobs: string[];
+  /** Total number of shares issues by this corporation */
+  bribeToRepRatio: number;
+  /** Amount of share owned */
+  baseMaxProducts: number;
+  /** Cooldown until shares can be sold again */
+  cityExpandCost: number;
+  /** Amount of acquirable shares. */
+  warehousePurchaseCost: number;
+  /** Price of the shares */
+  coffeeCost: number;
+  /** Fraction of profits issued as dividends */
+  materials: Record<string, materialInfo>;
+  /** Tax applied on your earnings as a shareholder */
+  products: Record<string, productInfo>;
+  /** Your earnings as a shareholder per second this cycle */
+  divisions: Record<string, divisionInfo>;
```

Comment:
Almost all the documentation comments are referencing the incorrect data, e.g. "Price of the shares" documentation for coffeeCost.

Reply:
hmm.. I thought I already fixed all those :/